README.md

```diff
@@ -1 +1,12 @@
-# python-uno
\ No newline at end of file
+# Python Uno
+
+An Uno clone built in Python 
+[Official Uno rules](https://www.unorules.com/)
```

Comment:
Small comment on the Markdown- you'll need to do a double enter to bring `[Official Uno rules]` to its own line- like this:

```
3  An Uno clone built in Python 
4  
5  [Official Uno rules](https://www.unorules.com/)
```

Not a requirement to fix, but if you want to change that it's an easy commit